client/v2/autocli/msg.go

```diff
@@ -183,5 +203,51 @@ func (b *Builder) BuildMsgMethodCommand(descriptor protoreflect.MethodDescriptor
 		cmd.SilenceUsage = true
 	}
 
+	// set gov proposal flags if command is a gov proposal
+	if options.GovProposal {
+		govcli.AddGovPropFlagsToCmd(cmd)
+		cmd.Flags().Bool(flags.FlagNoProposal, false, "Skip gov proposal and submit a normal transaction")
+	}
+
 	return cmd, err
```

Comment:
Nit but in this case, err == nil, making it explicit is important for code readability: `return cmd, nil`